docs/testplan/Chassis-fabric-test-plan.md

```diff
@@ -21,11 +21,74 @@ The scope of this test plan is as follows:
 
 The current SW design for fabric does not cover events like card insertion/removal or reboots. This test plan depends on fabric counter cli support (work in progress).
 
-# Test Setup
+## Test Setup
 
 These test cases will be run in the proposed [T2 topology](https://github.com/Azure/sonic-mgmt/blob/master/ansible/vars/topo_t2.yml). It is assumed that such a configuration is deployed on the chassis.
 
-tbinfo will be populated with the number of fabric links per forwarding ASIC that are expected to be up.
+These test cases will compare the following two sets of data on a chassis:
+* Expected fabric link status
+* Current fabric link status
+
+The following section describes how to store and process the expected fabric link status for testing.
+
+
+![](Img/Sonic_Fabric_Link_Testing_Proposal.png)
+
+The above diagram illustrates an example system under test. Every forwarding ASIC is connected to every fabric ASIC.
+
+The expected fabric link status is stored in the last section of testbed.yaml file, which is called fabric_link_toplogy. The expected status is provided per host and per ASIC, and the format is as follows:
```

Comment:
Please specify if this check is expected to be done on all line cards and sup as well.

Reply:
The check is done for both linecards and fabric cards on sup.  I also updated document with this information